src/nunavut/lang/properties.ini

```diff
@@ -277,3 +277,15 @@ named_values =
     true = true
     false = false
     null = null
+
+[nunavut.lang.docgen]
```

Comment:
I'm thinking this should be `nunavut.lang.html`. We could have a philosophical argument about this but I do believe this is consistent. We are generating a language here but it is not a machine language; it is a natural language and we use HTML to describe it. This would be consistent with other documentation generation targets like `nunavut.lang.pdf`.